### ISSUE TITLE
The performance of DeepGTT

### ISSUE THREAD

**Issue by learnurban** [boathit/deepgtt#6]
I have tested your code in the Didi of Chengdu dataset. However, the performance (MAE-6.12) is worse than DeepTTE(MAE-1.68). However, in your paper, you introduce your results better than DeepTTE. I check the code several times and don't find any errors in preprocessing and model. Can you provide the experimental code for Chengdu DIDI data?

**Comment by learnurban** [comment]
If you have time, please contact me. I can provide the entire code of my test for Chengdu DiDi data.

**Comment by learnurban** [comment]
Thank you for your reply. Maybe you can find the key of my worse results!

**Comment by boathit** [comment]
I have never conducted experiments on the Chengdu dataset and we only used the Harbin dataset in our paper. But as far as I know, the GPS points of the Chengdu dataset adopt a different coordinate system rather than the standard WGS-84, did you transform the coordinates before doing map matching?

**Comment by learnurban** [comment]
Yes, I transformed the Gaode GPS coordinate into standard WGS-84. I have tried many parameters of the xtsep and ystep. But the results of deepGTT are still poor. Also, I mean that your results of Deep GTT are better than DeepTTE in your paper, not Chengdu DIdi dataset.

**Comment by learnurban** [comment]
In other words, in your Harbin dataset, you mentioned that DeepTTE has poor performance. Do you know what causes DeepTTE to perform poorly in the Harbin dataset?

**Comment by boathit** [comment]
I will try to test the models on the Chengdu dataset and figure out why DeepGTT fails.

**Comment by learnurban** [comment]
Thank you very much! I look forward to your reply!
